src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-basic.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Collector with New Relic (basic setup)
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: Here is a simple Docker example to set up an OpenTelemetry collector.
+---
+
```

Comment:
This doc looks fine to me, but I do see that a SME has a few comments in it. Not sure if those have been resolved?

Reply:
@barbnewrelic, thanks, I'll work with the SME on this.